addons/account_peppol/models/res_company.py

```diff
@@ -14,21 +14,36 @@
     phonenumbers = None
 
 
+def _cc_checker(country_code, code_type):
+    return lambda endpoint: get_cc_module(country_code, code_type).is_valid(endpoint)
+
+
+def _re_sanitizer(expression):
+    return lambda endpoint: (res.group(0) if (res := re.search(expression, endpoint)) else None)
```

Comment:
hello :slightly_smiling_face: I think it should be endpoing here instead of `None` because with this, if there is no match, we just erase the value without any error or clarification for the user

Reply:
Corrected it and repushed it. However, it will some what neuter the sanitizion function.

However, your right that it will not give any errors and that might be an educational problem for the end user. But Peppol-ID connected to EAS is like Highlander, There Can Be Only One! ;-)... So anything that isn't following it is really wrong.

Reply:
that's true, but if there is no match, we will catch it in the check and then the user will at least see that it's a wrong value. I think that should be better :)